app/main.py

```diff
@@ -1,3 +1,42 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.loads(file.read())
```

Comment:
There's a dedicated method for file reading in `json` module. Use `json.load(file)`.